Resources/Private/Language/locallang_csh_media.xlf

```diff
@@ -6,8 +6,14 @@
 			<trans-unit id="type.description" xml:space="preserve">
 				<source>The type of the media element, can be video or audio.</source>
 			</trans-unit>
+			<trans-unit id="subtitles.description" xml:space="preserve">
+				<source>An optional subtitle file(s) for the video.</source>
```

Comment:
```suggestion
				<source>Optional subtitle files for the video.</source>
```

Reply:
I will also make some other changes, as the audio files may have subtitles too. somthing like this:

`<source>Optional subtitle files for the media file.</source>`